src/lib/core/datetime/calendar-locale.ts

```diff
@@ -0,0 +1,145 @@
+import {SimpleDate} from './simple-date';
+import {Injectable} from '@angular/core';
+
+
+const SUPPORTS_INTL_API = !!(new Date().toLocaleDateString);
```

Comment:
Safari 9 has `toLocaleDateString` but not the full `Intl` API.

Here's the check that modernizr uses: 
https://github.com/Modernizr/Modernizr/blob/master/feature-detects/intl.js